node-operators/network-node/tanssi/tanssi-node-systemd.md

```diff
@@ -0,0 +1,200 @@
+    ```
+
+4. Move the binaries to the folder:
+
+=== "Dancelight"
+
+    ```bash
+    mv ./tanssi-relay* /var/lib/dacelight-data
+    ```
+
+### Generate the Node Key {: #generate-node-key }
+
+To generate and store on disk the session keys that will be referenced on the start-up command, run the following command:
+
+=== "Dancelight"
+
+    ```bash
+    /var/lib/dancelight-data/tanssi-relay key generate-node-key --file /var/lib/dancelight-data/node-key
+    ```
+
+--8<-- 'text/node-operators/sequencers/onboarding/run-a-sequencer/generate-node-key-unsafe-note.md'
+
+### Create the Systemd Service Configuration File {: #create-systemd-configuration }
+
+The next step is to create the Systemd configuration file.
+
+You can create the file by running the following command:
+
+```bash
+sudo touch /etc/systemd/system/tanssi.service
+```
+
+Now you can open the file using your favorite text editor (vim, emacs, nano, etc) and add the configuration for the service, replacing the `INSERT_YOUR_TANSSI_NODE_NAME` tag with a human-readable name and `YOUR_IP_ADDRESS` with your public IP address. The name will come in handy for connecting the log entries and metrics with the node that generates them.
+
+=== "Dancelight"
+
+    ```bash
+    [Unit]
+    Description="Tanssi Node systemd service"
+    After=network.target
+    StartLimitIntervalSec=0
+
+    [Service]
+    User=tanssi_service
+    Type=simple
+    Restart=always
+    RestartSec=10
+    SyslogIdentifier=tanssi
+    SyslogFacility=local7
+    KillSignal=SIGHUP
+    LimitNOFILE=100000
+    ExecStart=/var/lib/dancelight-data/tanssi-relay --chain=dancelight \
+    --base-path=/var/lib/tanssi-data/ \
+    --node-key-file /var/lib/dancelight-data/node-key \
+    --rpc-port=9944 \
+    --prometheus-port=9615 \
+    --prometheus-external \
+    --name=INSERT_YOUR_TANSSI_NODE_NAME \
+    --listen-addr=/ip4/0.0.0.0/tcp/30333 \
+    --public-addr=/ip4/YOUR_IP_ADDRESS/tcp/30333 \
+    --state-pruning=archive \
+    --blocks-pruning=archive \
+    --database=paritydb \
+    --unsafe-rpc-external \
+    --telemetry-url='wss://telemetry.polkadot.io/submit/ 0'
+
+    [Install]
+    WantedBy=multi-user.target
+    ```
+
+### Run Flags {: #run-flags }
+
+The flags used in the `ExecStart` command can be adjusted according to your preferences and hardware configuration. The following ones are some of the most note-worthy:
+
+- `--state-pruning=archive` - Keeps all state data, which is necessary for historical state queries
+- `--blocks-pruning=archive` - Keeps all blocks, necessary for historical block data
+- `--database=paritydb` - Uses ParityDB as the database backend, which is optimized for RPC node performance
+- `--unsafe-rpc-external` - Allows external connections to the RPC server. This is required for the node to be accessible externally, but exposing RPC endpoints carries security risks. Ensure appropriate firewall and security measures are in place (see warning below)
```

Comment:
Update list please. Items left of hypen bold. Items right of hypen should not be capitalized. Example:

- **`--state-pruning=archive`** - keeps all state data, which is necessary for historical state queries